dockerfiles/evmapp/entrypoint.sh

```diff
@@ -198,6 +200,12 @@ if [ -z "${SCNODE_LOG_CONSOLE_LEVEL:-}" ]; then
 fi
 export SCNODE_LOG_FILE_LEVEL SCNODE_LOG_CONSOLE_LEVEL
 
+# History reset modifier status
+if [ "${SCNODE_HISTORY_RESET_MODIFIER_STATUS:-}" = "true" ]; then
+  HISTORY_RESET_MODIFIER_STATUS=true
+fi
```

Comment:
Maybe we could update the line 17 and modify line 204 in this way : 
Line 17 : 
HISTORY_RESET_MODIFIER_STATUS=""
Line 204 : 
if [ "${SCNODE_HISTORY_RESET_MODIFIER_STATUS:-}" = "true" ]; then
  HISTORY_RESET_MODIFIER_STATUS=true
else 
  HISTORY_RESET_MODIFIER_STATUS=false
fi
@paolocappelletti your approach works but in this way it could be more readable and we keep the same approach as the other variables (like ONLY_CONNECT_TO_KNOWN_PEERS="" etc)

Reply:
changed  a bit to make it similar to ONLY_CONNECT_TO_KNOWN_PEERS